src/status_im/ui/screens/chat/message/reactions.cljs

```diff
@@ -55,8 +55,7 @@
                               (fn []
                                 (reset! actions nil)
                                 (reset! visible false)
-                                (picker-on-close))
-                              reaction-picker/animation-duration))
+                                (picker-on-close))))
```

Comment:
`setTimout` without any duration, looks interesting solution. Maybe you can pass 0 as the second parameter instead of removing it.